src/System.Windows.Forms/src/System/Windows/Forms/AccessibleObject.cs

```diff
@@ -2702,6 +2185,231 @@ private static IntPtr GetAddressOfVariantAtIndex(IntPtr variantArrayPtr, uint in
+                return SafeGetValue(() => _systemIAccessible.get_accDescription(childID));
+            }
+
+            public object accRole => SafeGetValue(() => _systemIAccessible.accRole);
+
+            public object get_accRole(object childID)
+            {
+                return SafeGetValue(() => _systemIAccessible.get_accRole(childID));
+            }
+
+            public object accState => SafeGetValue(() => _systemIAccessible.accState);
+
+            public object get_accState(object childID)
+            {
+                return SafeGetValue(() => _systemIAccessible.get_accState(childID));
+            }
+
+            public string accHelp => SafeGetValue(() => _systemIAccessible.accHelp);
+
+            public string get_accHelp(object childID)
+            {
+                return SafeGetValue(() => _systemIAccessible.get_accHelp(childID));
+            }
+
+            public int get_accHelpTopic(out string pszHelpFile, object childID)
+            {
+                string helpFile = null;
+                int result = SafeGetStructValue(() => _systemIAccessible.get_accHelpTopic(out helpFile, childID), -1);
+                pszHelpFile = helpFile;
+                return result;
+            }
+
+            public string accKeyboardShortcut => SafeGetValue(() => _systemIAccessible.accKeyboardShortcut);
+
+            public string get_accKeyboardShortcut(object childID)
+            {
+                return SafeGetValue(() => _systemIAccessible.get_accKeyboardShortcut(childID));
+            }
+
+            public object accFocus => SafeGetValue(() => _systemIAccessible.accFocus);
+
+            public object accSelection => SafeGetValue(() => _systemIAccessible.accSelection);
+
+            public string accDefaultAction => SafeGetValue(() => _systemIAccessible.accDefaultAction);
+
+            public string get_accDefaultAction(object childID)
+            {
+                return SafeGetValue(() => _systemIAccessible.get_accDefaultAction(childID));
+            }
+
+            private protected TReturn SafeGetValue<TReturn>(Func<TReturn> getFunction) where TReturn : class
+            {
+                if (_systemIAccessible == null || getFunction == null)
+                {
+                    return null;
+                }
+
+                try
+                {
+                    return getFunction();
+                }
+                catch (COMException e) when (e.ErrorCode == (int)HRESULT.DISP_E_MEMBERNOTFOUND)
+                {
+                    // System IAccessible is not found.
+                }
+                catch (ArgumentException)
+                {
+                    // Argument exception can be thrown in case main system IAccessible cannot be gotten
+                    // with MEMBERNOTFOUND and then all children (ChildId > 0) also cannot be gotten.
+                }
+
+                return null;
+            }
+
+            private TReturn SafeGetStructValue<TReturn>(
```

Comment:
Is there a specific reason not to name it 
```suggestion
            private TReturn GetValue<TReturn>(
```

Reply:
Renamed. Thanks!

Reply:
Still there...

Reply:
There was `SafeGetStructValue` replaced with `GetStructValue`. In the latest commit renamed to just `GetValue`.